source/02_algo/model_based_rl_eng.rst

```diff
@@ -0,0 +1,147 @@
+Model-Based Reinforcement Learning
+######################################
+
+
+Model-Based Reinforcement Learning (Model-Based RL) is an important branch of reinforcement learning. The agent learns an environmental dynamics model through information interacting with the environment, and then uses the model to generate data to optimize action strategies, or use the model for planning.
```

Comment:
Model-Based Reinforcement Learning (Model-Based RL) is an important branch of reinforcement learning. The agent learns a **_dynamics_** model _**by interacting with the environment**_ and then uses the model to generate data to optimize policy or use the model for planning.